tox.ini

```diff
@@ -35,4 +35,4 @@ deps =
 [testenv:flake8]
 deps = flake8
 skip_install = true
-commands = flake8 setup.py linkcheck {posargs}
+commands = flake8 linkchecker setup.py linkcheck {posargs}
```

Comment:
Having specified `filename =` in the config file, do we have to list them here, again?

Actually this list is different: linkcheck is a directory, not a filename.

Reply:
This appears to override the `filename =` i.e. it doesn't scan tests or scripts. Maybe we should scan tests?

Reply:
I would like to scan all Python files.

(My vim has flake8 error highlighting via vim-ale, so I see those warnings all the time.  This incentivizes me to keep my code flake8-clean.)

Reply:
OTOH if the choice is between getting linkchecker/ flake8-clean today and making the CI job required vs. getting linkchecker/ and tests/ flake8-clean next year .... it may be better to settle for less but sooner.

I haven't looked at flake8 warnings in tests/.

Reply:
It's almost all formatting, plus a few unused variables, bare excepts.